src/test/java/pages/ProductsPage.java

```diff
@@ -0,0 +1,55 @@
+package pages;
+
+import com.codeborne.selenide.SelenideElement;
+import helpers.ConfigurationManager;
+import io.qameta.allure.Step;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import static com.codeborne.selenide.Selenide.*;
+
+public class ProductsPage extends AbstractPage {
+
+    private final String PAGE_URL = ConfigurationManager.getProperty("base.url") + "/inventory.html";
```

Comment:
PAGE_URL = ConfigurationManager.getProperty("base.url") exists on AbstractPage. Please reuse it